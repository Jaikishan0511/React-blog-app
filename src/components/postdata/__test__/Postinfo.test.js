import React from "react";
import renderer from "react-test-renderer";

import Postinfo from "../index";
describe("Postinfo", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Postinfo />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
