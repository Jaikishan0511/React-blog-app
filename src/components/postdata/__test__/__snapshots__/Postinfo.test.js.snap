// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Postinfo snapshot renders 1`] = `
<div>
  <div>
    <img
      alt="loading..."
      src="/Pacman-1s-200px.gif"
    />
    ;
  </div>
</div>
`;
